app/models/manageiq/providers/openstack/cloud_manager/vm/operations/relocation.rb

```diff
@@ -18,7 +20,14 @@ def raw_live_migrate(options = {})
     block_migration  = options[:block_migration]  || false
     disk_over_commit = options[:disk_over_commit] || false
     with_provider_connection do |connection|
-      connection.live_migrate_server(ems_ref, hostname, block_migration, disk_over_commit)
+      begin
+        connection.live_migrate_server(ems_ref, hostname, block_migration, disk_over_commit)
+      rescue => ex
+        error_message = parse_error_message_from_fog_response(ex.to_s)
+        Notification.create(:type => :vm_cloud_live_migrate_error, :options => {:instance_name => name, :error_message => error_message})
```

Comment:
Shouldn't we re-throw the exception?

So, the `raw_power_state` is not modified? (or any other side-effects)